docs/getting_started/index.md

```diff
@@ -116,62 +116,112 @@ While we've used Taipy's Python APIs to handle our scenario, it usually works se
 graphical interface, also created using Taipy, to provide a more user-friendly experience. 
 Here's a simple GUI set up for our "Hello World" scenario:
 
-```python linenums="1"
-import taipy as tp
+!!! example "GUI creation"
 
-# Previous configuration of scenario
-...
+    === "Markdown"
 
-page = """
-Name: <|{input_name}|input|>
-<|submit|button|on_action=submit_scenario|>
+        Taipy pages can be defined in multiple ways: Markdown, Html or Python. *page* is the Markdown representation of the page.
 
-Message: <|{message}|text|>
-"""
+        ```python linenums="1"
+        import taipy as tp
 
-input_name = "Taipy"
-message = None
+        # Previous configuration of scenario
+        ...
 
+        page = """
+        Name: <|{input_name}|input|>
+        <|submit|button|on_action=submit_scenario|>
 
-def submit_scenario(state):
-    state.scenario.input_name.write(state.input_name)
-    state.scenario.submit(wait=True)
-    state.message = scenario.message.read()
+        Message: <|{message}|text|>
+        """
 
+        input_name = "Taipy"
+        message = None
 
-if __name__ == "__main__":
-    tp.Core().run()
-    scenario = tp.create_scenario(scenario_cfg)
-    tp.Gui(page).run()
-```
 
-[Get the whole code](./hello_world.py){: .tp-btn target='blank'}
+        def submit_scenario(state):
+            state.scenario.input_name.write(state.input_name)
+            state.scenario.submit(wait=True)
+            state.message = scenario.message.read()
 
-Now, let’s explain the key elements of this code:
 
-```python
-import taipy as tp
+        if __name__ == "__main__":
+            tp.Core().run()
+            scenario = tp.create_scenario(scenario_cfg)
+            tp.Gui(page).run()
+        ```
 
+        [Get the whole code](./hello_world_md.py){: .tp-btn target='blank'}
+
+        Now, let’s explain the key elements of this code:
+
+        ```python
+        import taipy as tp
+
+
+        page = """
+        Name: <|{input_name}|input|>
+        <|submit|button|on_action=submit_scenario|>
+        Message: <|{message}|text|>
+        """
+        ```
+
+    === "Python"
+
+        Taipy pages can be defined in multiple ways: Markdown, Html or Python. *page* is built through the Taipy Gui Builder.
+
+        ```python linenums="1"
+        import taipy as tp
+        import taipy.gui.builder as tgb
+
+        # Previous configuration of scenario
+        ...
+
+        with tgb.Page() as page:
+            tgb.text("Name:")
+            tgb.input("{input_name}")
```

Comment:
I suspect that, when https://github.com/Avaiga/taipy/pull/635 is merged, that can be written:
`tgb.input(input_name)`

If I were you, I would give this a shot.

Reply:
Wow, I didn't know about this one. This is impressive

Reply:
That is for 3.1